server/data/data-cassandra/pom.xml

```diff
@@ -119,6 +119,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-configuration2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
```

Comment:
I don't see where using it?

Reply:
Huh, I remember I had a problem running some test without it, but I just re-checked and now everything works. Strange. I will remove this again.